app/models/form.rb

```diff
@@ -83,6 +83,15 @@ def next_page_redirect_path(page, log, current_user)
     end
   end
 
+  def next_page_interruption_screen?(page, log, current_user)
+    nxt_page_id = next_page(page, log, current_user)
+    nxt_page = get_page(nxt_page_id)
+
+    return unless nxt_page
+
+    nxt_page.questions.map(&:type).uniq.first == "interruption_screen"
```

Comment:
Highly unlikely that we will have more than 1 question on the page for this but we could do something like:
`nxt_page.questions.any? { |question| question.type == "interruption_screen" }`